### PR TITLE
Env Vars to identify image #207

### PR DESCRIPTION
**Pull request by gregw**
Set env vars to identify image #207

### CODE REVIEW

**Comment by gregw** [comment]
I signed it!

File: appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java [hunk 1]
```diff
@@ -48,6 +48,8 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
```
Comment (aozarov) [inline appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java]: This changes belong to #257, right?

File: jetty9-compat/src/main/docker/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jetty9:@@image.label@@
+FROM jetty9:${docker.label.prefix}${jetty.version}
+
+# Create env vars to identify image
+ENV GAE_IMAGE_NAME ${project.artifactId}
+ENV GAE_IMAGE_LABEL ${docker.label.prefix}8-jre
```
Comment (aozarov) [inline jetty9-compat/src/main/docker/Dockerfile]: should that be the same as the openjdk8 Dockerfile?

**Comment by aozarov** [comment]
Looks good! One change seems to belong to a different PR and I have one question regarding the compat label. @ludoch are you OK with these env values?

File: appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java (hunk 1 above)
Comment (gregw) [inline appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java]: Ah yes, I branched off #257.   I've merged with the latest version of that, but can remake as a separate PR if you prefer.

File: jetty9-compat/src/main/docker/Dockerfile (hunk 2 above)
Comment (gregw) [inline jetty9-compat/src/main/docker/Dockerfile]: No, they had different label content.  I've just copied the existing variables from the pom.xml to the Dockerfile

File: appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java (hunk 1 above)
Comment (aozarov) [inline appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java]: Yes, lets leave the 2 separate.

File: jetty9-compat/src/main/docker/Dockerfile (hunk 2 above)
Comment (aozarov) [inline jetty9-compat/src/main/docker/Dockerfile]: In the pom before the change, doesn't it say "token="@@image.label@@" value="${docker.label.prefix}${jetty.version}"/> ? Am I confused?

Comment (gregw) [inline jetty9-compat/src/main/docker/Dockerfile]: The names/labels assigned to the images in the pom.xmls are:

- `openjdk8:${docker.label.prefix}8-jre`
- `jetty9:${docker.label.prefix}${jetty.version}`
- `jetty9-compat:${docker.label.prefix}${project.version}`

The images used by each docker file are:
- `gcr.io/google_appengine/base`
- `openjdk8:${docker.label.prefix}8-jre`
- `jetty9:${docker.label.prefix}${jetty.version}`

So I think they line up correctly.

Comment (aozarov) [inline jetty9-compat/src/main/docker/Dockerfile]: I still don't see why GAE_IMAGE_LABEL for jetty9-compat is ${docker.label.prefix}**8-jre** but we can talk about it in our meeting soon.

Comment (gregw) [inline jetty9-compat/src/main/docker/Dockerfile]: Oh sorry - I missed what you were saying.  I was looking at the labels below that level! fixed now

File: appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java (hunk 1 above)
Comment (aozarov) [inline appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java]: Do you plan to take it out from this PR and leave it as part of #257?

**Comment by aozarov** [comment]
Other than taking out the #257 changes all look good to me.

**Comment by gregw** [comment]
I can take out the #257 changes, but if that PR is ready to merge, perhaps it is simplest to merge #257, I will then update this PR and it will just include those changes as part of the merge rather than as part of this PR.

**Comment by aozarov** [comment]
Sounds good. I think 2 minor changes are needed for #257 and then you can merge it and update this PR if that is easier.

**Pull request merged**